docs/meshstack.osb-mesh-specifics.md

```diff
@@ -0,0 +1,127 @@
+
+## meshMarketplace Profile
+
+The OSB API Spec allows platforms to define various extensions as part of a [Profile](https://github.com/openservicebrokerapi/servicebroker/blob/v2.14/profile.md)
+
+This section outlines the implementation-specific extensions of the meshMarketplace.
+
+### Context Object
+
+meshMarketplace defines its own context objects for service provisioning requests. For the convenience of the service broker, the meshMarketplace delivers the following information:
+
+```json
+{
+  "platform": "http://{mesh-hostname}/serviceRegistry/location/1",
+  "customer_id": "testCustomer",
+  "project_id": "testProject",
+  "auth_url": "https://{mesh-hostname}/auth/realms/meshfed/protocol/openid-connect/auth?client_id=1d4ad6d8-dfaa-4913-9c12-fd64b42a5c8d&response_type=code&redirect_uri={redirect_uri}&nonce={nonce}&state={state}",
+  "token_url": "https://{mesh-hostname}/auth/realms/meshfed/protocol/openid-connect/token",
+  "permission_url": "http://{mesh-hostname}/serviceInstances/c48d065b-a123-4a1e-8021-2965928d022d/permissions"
+}
+```
+
+It is recommended that Service Brokers store this information as it allows operators to more easily identify links between service instances and projects when handling support requests or monitoring service operation.
+
+### Catalog Metadata
+
+The OSB API Spec defines free-form metadata fields for service instances and plans. Even though not standardized, there are established conventions around their use. In order to have their service properly rendered on the meshMarketplace User Interface, Service Brokers must stick to the [OSB Profile metadata conventions](https://github.com/openservicebrokerapi/servicebroker/blob/master/profile.md#service-metadata).
+
+#### Cost Information
+
+The OSB profile also contains properties to provide [cost information](https://github.com/openservicebrokerapi/servicebroker/blob/master/profile.md#cost-object) via the OSB catalog. Besides showing this information to the user of the meshMarketplace, this information is also used in the [meshMetering](meshstack.metering-meshmarketplace.md) component. That means cost information provided via the OSB catalog will be used to calculate costs for used services my meshMetering. In order to successfully parse the cost information provided via the catalog, the following rules have to be considered, as the OSB Profile is not specific enough to use the cost information for automated pricing of service usages:
+
+- unit must be one of: `HOURLY, WEEKLY, MONTHLY, YEARLY, MB, GB`
+- currently only one cost component can be set per service plan. Multiple different cost components per plan are not supported yet. If provided anyway, meshMetering will only use one of the defined cost components (most likely the last one).
+- currently only `eur` is supported as a currency
+
```

Comment:
I think this could use an example snippet.

Reply:
the snippet is mentioned in the osb profile docs, so i won't add it here for now